content/company-info-and-process/diversity-equity-and-inclusion/queer.md

```diff
@@ -0,0 +1,27 @@
+# Cohort for LGBTQUIA2s+
+
+_Known informally by the name of the group’s private Slack channel, #queer._
+
+## What is the cohort?
+
+The cohort is an employee-led opt-in group for fostering inclusivity, community, and connectedness amongst members of the LGBTQIA2S+ community. The cohort fits the traditional definition of an “Employee Resource Group (ERG)” (a network of employees who share certain characteristics, background, or experiences that is sponsored by the organization they work for), but for the purposes of keeping the group’s methodology light and unrestrictive, we prefer in most settings to refer to it as a “cohort” instead.
```

Comment:
Just a suggestion, maybe provide a link to what LGBTQIA2S+ stands for, since some folks (particularly non-American readers) may not all be aware or know what the "longer" form means?

Reply:
Added it in parentheses since the links I could find were all meh. Good catch!